apps/examples/src/examples/basic/BasicExample.tsx

```diff
@@ -1,10 +1,22 @@
-import { Tldraw } from 'tldraw'
+import { useState } from 'react'
+import { Tldraw, TldrawOptions } from 'tldraw'
 import 'tldraw/tldraw.css'
 
 export default function BasicExample() {
```

Comment:
Is this meant to stay on BasicExample? Seems like the changes should be a different example.

Reply:
oh yeah sorry this wants cleaning up!